fluent-schema.d.ts

```diff
@@ -0,0 +1,15 @@
+declare module "fluent-schema" {
+
+  function FluentSchema(opt?:SchemaOptions): FluentSchema
+
+  interface SchemaOptions {
+    schema: object,
+    generateIds: boolean
+  }
+
+  interface FluentSchema {
+    prop: (name: string, props?: object) => FluentSchema
+  }
```

Comment:
I think this is missing some methods, such as `required`.

Reply:
yes, it's just a placeholder consider this PR not complete. a lot it's missing

Reply:
I will add the tests as well